tests/testthat/test-clean_redcap.R

```diff
@@ -45,8 +45,8 @@ test_that("extract_nonrepeat_table tibble contains expected columns and data typ
   expect_character(out$text)
```

Comment:
I think you have to repeat this section on a data set in which `raw_or_label == label`.